ballerina/tests/negative.bal

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/test;
+import ballerina/lang.'string as langstring;
+
+@test:Config {}
+isolated function testFileNotFound() {
+    string filePath = TEMP_DIR + "unknown.txt";
+    var err = openReadableFile(filePath);
+
+    if err is Error {
+        test:assertTrue(langstring:includes(err.message(), "no such file or directory:", 0));
+        test:assertTrue(langstring:includes(err.message(), "unknown.txt", 0));
```

Comment:
```suggestion
        test:assertTrue(err.message().includes("no such file or directory:"));
        test:assertTrue(err.message().includes("unknown.txt"));
```

Reply:
We may also like the above easily with lang.string import

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/400/commits/551dea9487bb7b9e1c9bd608aef5fdaf45fe21ed